src/App.js

```diff
@@ -1,11 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 import './App.css';
-
+import Pokemon from "./assets/Pokemon";
+import Button from "./assets/Button";
+//TEST
 function App() {
 
+    const [pokemon, setPokemon] = useState(null);
+    const [url, setUrl] = useState('https://pokeapi.co/api/v2/pokemon');
+    const [loading, toggeleLoading] = useState(false);
+    const [error, setError] = useState(false);
+
+    useEffect(() => {
+
+        const token = axios.CancelToken.source();
+
+        async function fetchData() {
+            toggeleLoading(true);
+            setError(false);
+
+            try {
+                const result = await axios.get(url);
+                setPokemon(result.data);
+                toggeleLoading(false);
```

Comment:
`toggleLoading(false)` kan ook in één keer ná het try/catch blok gezet worden. Of het nou gelukt is of niet, loading moet sowieso weer op `false`.